docs/api/specs/webConnectionProtocol.md

```diff
@@ -0,0 +1,382 @@
+---
+id: webConnectionProtocol
+sidebar_label: Web Connection Protocol
+title: FDC3 Web Connection Protocol (next)
```

Comment:
Minor point, but in this title we describe this as "FDC3 Web Connection Protocol", but in the equivalent title in desktopAgentCommunicationProtocol.md it simply says "Desktop Agent Communication Protocol" - with no "FDC3" qualifier at the start.

Unless there's a specific reason for this, for consistency perhaps it would be better for both pages to use the "FDC3" qualifier, or neither page to do so.

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85